app/db/storage_assumptions.md

```diff
@@ -0,0 +1,35 @@
+# Storage assumptions
+
+## Method
+
+- Reviewed disk usage in postgres database and backup after running env-canada and noon-forecast cronjob.
+- Reviewed current database backup files to establish compression ratio.
+
+## Assumptions
+
+- Optimization by a factor of 10 on env-canada job should be feasable.
+- Compression ratio for backup will remain consistent.
+- Forecast data has very little room for optimization.
+
+Refer to storage_assumptions spreadsheet for further assumptions and details of calculations.
+
+## Scenario 1 : current
+
+Assuming no changes to the application and how it is run as of 2020 August 11.
+
+- Storage: Run out of space in 28.5 years, sufficient storage available.
+- Backup: Run out of space in 0.5 years. Require at least 11 GB, ideally 42 GB.
+
+## Scenario 2 : projected
+
+Assuming no changes in how data is stored, adding additional fields and adding the high resolution regional model.
+
+- Storage: Run out of space in 0.8 years. Required 1.2TB.
+- Backup: Run out of space in 0.0 years. Require at least 669GB, ideally 1.4TB.
```

Comment:
how imminently will we run out of space? like in a few weeks' time?

Reply:
With what we're storing right now, we have about half a year. The moment we add the other fields, it turns into days. There is some wiggle room though, right now there are 6 daily backups, 4 weekly backups and a monthly backup - we can just have less backups.